barman/infofile.py

```diff
@@ -731,6 +731,26 @@ def __init__(self, server, info_file=None, backup_id=None, **kwargs):
                 e,
             )
 
+    @property
+    def is_incremental(self):
+        """
+        Only checks if the backup_info is an incremental backup
+
+        :param barman.infofile.BackupInfo backup_info: The object we want to
+            do the check.
```

Comment:
```suggestion
        :return bool: ``True`` if this backup is incremental, ``False`` otherwise.
```

There is no `backup_info` param, and we are missing the return value.